components/dmx512/dmx512.cpp

```diff
@@ -55,14 +55,24 @@ void DMX512::write_channel(uint16_t channel, uint8_t value) {
   this->update_ = true;
 }
 
+uint8_t DMX512::get_channel_value(uint16_t channel) {
+  return this->device_values_[channel];
+}
+
 void DMX512Output::set_channel(uint16_t channel) {
   this->channel_ = channel;
   if(this->universe_) {
     this->universe_->set_channel_used(channel);
   }
 }
 
+float DMX512Output::get_state() {
+  state = this->universe_->get_channel_value(this->channel_) / 255.0;
```

Comment:
Use `this` pointer when referencing member variables.

Reply:
You mean like this I guess ? My bad I havent tested that one specifically.
``` C++ 
this->state = ... 
```
I guess we could drop
``` C++
float DMX512Output::get_state() {
  state = this->universe_->get_channel_value(this->channel_) / 255.0;
  return state;
}
```
considering that we are setting `this->state` in `DMX512Output::write_state`